scylla/src/frame/frame_errors.rs

```diff
@@ -5,7 +5,7 @@ use thiserror::Error;
 
 #[derive(Error, Debug)]
 pub enum FrameError {
-    #[error("Type parsing failed")]
+    #[error("{0}")]
```

Comment:
Would `#[error(transparent)]` work here instead? I think it will be more explicit that way, and probably slightly more efficient. More info on [this page](https://docs.rs/thiserror/1.0.26/thiserror/).

Reply:
Good idea, done